Problem1.py

```diff
@@ -0,0 +1,101 @@
+# Design Hashmap
+''' Time Complexity : O(hashmapsize/bucketsize) for put, get remove functions
+// Space Complexity : O(hashmapsize+Bucketsize) 
+// Did this code successfully run on Leetcode : YES
+// Any problem you faced while coding this : Took some time to understand the concept and key hashing'''
+
+class Node:# linked list to create buckets
+    def __init__(self,data):
+        self.data = data # data contains both key and value
+        self. next = None 
+        
+class MyHashMap:
+    ''' 1. find hash key _hash function -  criteria should be min numbe rof collision.           h(k) = k mod m. let m be size = 10.
+    2. Create a fixed size hash map 
+    3. put - insertion - find the block using hash key and then insert the value          under hashed key as a linked list node
+    4. get - find the block using hash key and find the element from linked list
+    5. remove - find the block using hash key and remove the element fro linked list''' 
+
+    def __init__(self):
+        """
+        Initialize your data structure here.
+        """
+        #hashmap of fixed size - array
+        self.hashmap = [None]*1000
+        
+    def _hash(self,data):
+        return data%10
```

Comment:
You have a bucket of size 1000 while you're actually using only the first 10 blocks of it. Why is that?